janusgraph-test/src/test/java/org/janusgraph/graphdb/log/StandardLogProcessorFrameworkTest.java

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.graphdb.log;
+
+import org.easymock.EasyMockSupport;
```

Comment:
I'm not much a fan of easymock. I wouldn't this easy possible to do with mockito

Reply:
Sure, I will change it to mockito

Reply:
Changed to mockito